common/install_cpython.sh

```diff
@@ -64,7 +65,13 @@ function build_cpython {
     check_var $py_ver
     check_var $PYTHON_DOWNLOAD_URL
     local py_ver_folder=$py_ver
-    wget -q $PYTHON_DOWNLOAD_URL/$py_ver_folder/Python-$py_ver.tgz
+    # Only b2 version of 3.12 is available right now
```

Comment:
1. What's a b2 version?
2. Should this be:
```suggestion
    # Only b2 version of 3.13 is available right now
```

Reply:
Actually, seems like we're not using the b2 version?

Reply:
yes true. The comment is not correct. Let me remove it